build/js/live-editor.output_pjs.js

```diff
@@ -1322,12 +1339,186 @@ window.PJSOutput = Backbone.View.extend({
         return this;
     },
 
+    initializeWorkers: function(options) {
+        // Note: all messages are stringified using JSON.stringify before
+        // being passed to the workers via postMessage because it's faster
+        // http://jsperf.com/web-worker-json-vs-string/13
+        
+        // then pass the promise in so that this class that pass
+        // those dependencies to workers when it resolves
+        // TODO(kevinb7) instantiate one of each worker before it's needed 
+        // this will avoid the initial lag seen when manipulating variables with the slider
+        this.workersPromise = $.get(options.workersDir + "deps.json").then(function(data) {
+            var jshintDeps = {
+                "es5-shim.js": data["es5-shim.js"],
+                "jshint.js": data["jshint.js"],
+                "underscore.js": data["underscore.js"]
+            };
+
+            // TODO(kevinb7) polyfill vendor prefixed version to this
+            // TODO(kevinb7) fallback to data URI on IE9 (or just use a real URL)
+            var jshintBlob = new Blob([data["jshint-worker.js"]], { type: 'application/javascript' });
+            this.hintWorker = new PooledWorker(
+                URL.createObjectURL(jshintBlob),
```

Comment:
The workers themselves are included in deps.json.  createObjectURL can create page-specific? URLs from Blobs and Files.  This needs to be made more robust see lines 1358 and 1359 for TODOs.